rnd/autogpt_server/autogpt_server/blocks/__init__.py

```diff
@@ -1,9 +1,10 @@
 from autogpt_server.blocks import sample, reddit, text, ai
+from autogpt_server.blocks.wikipedia import GetWikipediaSummary
 from autogpt_server.data.block import Block
 
 AVAILABLE_BLOCKS = {
     block.id: block
     for block in [v() for v in Block.__subclasses__()]
 }
 
-__all__ = ["ai", "sample", "reddit", "text", "AVAILABLE_BLOCKS"]
+__all__ = ["ai", "sample", "reddit", "text", "AVAILABLE_BLOCKS", "GetWikipediaSummary"]
```

Comment:
from autogpt_server.blocks import wikipedia

and expose wikipedia instead of GetWikipediaSummary